Themis/ViewModels/Assessment/AssessmentViewModel.swift

```diff
@@ -47,23 +142,46 @@ class AssessmentViewModel: ObservableObject {
     }
 
     @MainActor
-    func getSubmission(id: Int) async {
+    func getSubmission(submissionId: Int) async {
+        guard !readOnly else {
```

Comment:
Will this ever be called if this assumption is incorrect? I would feel like it shouldn't be => it should throw because the expectation is violated